packages/win-error/win-error.0.3/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+name: "win-error"
+maintainer: "dave@recoil.org"
+version: "0.3"
+authors: [ "David Scott" ]
+license: "ISC"
+homepage: "https://github.com/djs55/ocaml-win-error"
+dev-repo: "https://github.com/djs55/ocaml-win-error.git"
+bug-reports: "https://github.com/djs55/ocaml-win-error/issues"
+
+build: [
+  [ "jbuilder" "subst"] {pinned}
+  [ "jbuilder" "build" "-p" name "-j" jobs ]
+]
+
+build-test:[
+  [ "jbuilder" "subst"] {pinned}
+  [ "jbuilder" "runtest" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "base-bytes"
+  "base-unix"
+  "jbuilder" {build & >="1.0+beta10"}
+]
+available: [ ocaml-version >= "3.12.1" ]
```

Comment:
For the record this is not really accurate because `jbuilder` requires OCaml >= 4.02.3